### ISSUE TITLE
Cannot specify the input file

### ISSUE THREAD

**Issue by lethal233** [Anjok07/ultimatevocalremovergui#1008]
**Symptom**
As is shown in the screenshot, I can set up the output directory but cannot select input files

**Runtime environment**
Windows 11 + Nvidia 3080

![image](https://github.com/Anjok07/ultimatevocalremovergui/assets/47763046/efd17607-43e1-45ac-aadc-108baa2f8f24)

**Comment by ChrisTG742** [comment]
So if you click on the button "Select Input" on the left nothing happens?
The UX is a bit awkward here, the "folder symbol" on the right only lets you open the folder of the source file.

**Comment by lethal233** [comment]
> So if you click on the button "Select Input" on the left nothing happens? The UX is a bit awkward here, the "folder symbol" on the right only lets you open the folder of the source file.

No, I cannot click "select input". I can only click the text field and the file explorer icon on the right, but the file explorer will not be opened (it was always loading)
